homeassistant/components/switch/pilight.py

```diff
@@ -0,0 +1,216 @@
+"""
+
+Support for Pilight Switches.
+"""
```

Comment:
Hey, thank you for your help. A lot of the things are new for me. I add on the beginning a example for the entry in the configuration. But i have no plan, how i can set input variable as mandatory values. Maybe you can help me. 
```python 
"""
Support for Pilight Switches.

configuration example entry:
switch:
  platform: pilight
  ip: 127.0.0.1
  port: 5000
  switches:
    kitchen_desklight:    #name of the device in pilight
      name: desklight     #name in the home-assistant frontend
    kitchen_sink:
      name: sinklight
"""
```